RXSwiftTest/Model/User.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  User.swift
+//  RXSwiftTest
+//
+//  Created by Sunith B on 30/08/19.
+//  Copyright © 2019 Sunith B. All rights reserved.
+//
+
+import UIKit
+
+class User : Codable {
+    
+    static let shared = User()
+    
+    //MARK: - Intance Variables
+    var token : String
+    var balance : String
+    var currency : String
+    var loggedIn : Bool
+    var transactions : [Transaction]
+    
+    private init() {
+        token = "Bearer nukpymirqrtmykltocgi"
+        balance = ""
```

Comment:
If the balance property is empty is can be nil. Also Balance should be a Float or Double